app/code/Magento/NewRelicReporting/Plugin/HttpPlugin.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\NewRelicReporting\Plugin;
+
+use Magento\Framework\App\Bootstrap;
+use Magento\Framework\App\Http;
+use Magento\NewRelicReporting\Model\Config;
+use Magento\NewRelicReporting\Model\NewRelicWrapper;
+
+class HttpPlugin
+{
+    /**
+     * @var Config
+     */
+    protected $config;
+
+    /**
+     * @var NewRelicWrapper
+     */
+    protected $newRelicWrapper;
```

Comment:
In general looks good, btw new properties should be private instead of protected. Sorry for missing if prev. time

Reply:
Just wondering - even new properties to new classes?  I understand the BC concerns for existing classes (2.2.0 lacking something that 2.1.11 has, etc.), but does it help for new classes too in some way?

Reply:
According to technical guidelines: http://devdocs.magento.com/guides/v2.1/coding-standards/technical-guidelines/technical-guidelines.html

2.7. All non-public properties and methods SHOULD be private.

Reply:
@ihor-sviziev this is done